app/src/main/java/io/apicurio/registry/rest/v2/GroupsResourceImpl.java

```diff
@@ -214,6 +213,10 @@ public void updateArtifactMetaData(String groupId, String artifactId, EditableMe
         requireParameter("groupId", groupId);
         requireParameter("artifactId", artifactId);
 
+        if (data.getProperties() != null) {
+            data.getProperties().forEach((k,v) -> requireParameter("property value", v));
+        }
```

Comment:
This means that each property must have a name **and** a value, yes?  I was thinking it might be more powerful if we supported both.  So users could do a search for artifacts that had the property defined, regardless of the value of that property.  And **also** of course users can search for artifacts where the property has a specific value.  wdyt?

Reply:
sounds reasonable easy to implement

Reply:
Yes my bad.  I assumed the wrong context on this.